package.json

```diff
@@ -21,7 +21,8 @@
   },
   "ava": {
     "require": [
-      "babel-register"
+      "babel-register",
+      "babel-polyfill"
```

Comment:
Babel core slightly changed and is not injecting `runtimeGenerator` for async functionality by default. It is necessary to either use `babel-polyfill` or `babel-plugin-transform-runtime` to inject proper polyfill. https://babeljs.io/docs/plugins/transform-runtime/

This is required to ava run tests asynchronously.